### PR TITLE
NXP-26242: take enrichers into account in nuxeo-dropzone

### PR DESCRIPTION
**Pull request by ataillefer**

### CODE REVIEW

**Comment by guirenard** [comment]
:+1:

**Comment by nmpcunha** [comment]
👍

**Comment by guirenard** [comment]
see https://github.com/nuxeo/nuxeo-web-ui/commit/62f9b31970141b55f4a59bb685bd71eb5ed5963d

File: elements/nuxeo-dropzone/nuxeo-dropzone.html [hunk 1]
```diff
@@ -383,6 +389,15 @@
         return (!this.immediate && this.blobList) || !this.hasFiles;
       },
 
+      _computeEnrichers: function() {
+        var enrichers = Nuxeo.UI && Nuxeo.UI.config && Nuxeo.UI.config.enrichers;
```
Comment (guirenard) [inline elements/nuxeo-dropzone/nuxeo-dropzone.html]: useless or

File: elements/nuxeo-dropzone/nuxeo-dropzone.html [hunk 2]
```diff
@@ -383,6 +389,15 @@
...
+        return {
+          document: ['preview'],
+          blob: (Nuxeo.UI && Nuxeo.UI.config && Nuxeo.UI.config.enrichers && Nuxeo.UI.config.enrichers.blob)
```
Comment (guirenard) [inline elements/nuxeo-dropzone/nuxeo-dropzone.html]: (enrichers && enrichers.blob)

**Comment by guirenard** [comment]
:+1:

**Comment by nelsonsilva** [comment]
👍

**Pull request merged**